scripts/github-last-contribution.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+import argparse
+import urllib.request
+from bs4 import BeautifulSoup
+'''
```

Comment:
This comment should probably come right after the shebang (#!/usr/bin/env python).